src/partials/page-additional-icons.hbs

```diff
@@ -0,0 +1,6 @@
+<a class="page-icon" href="{{{edit2var page.editUrl "history"}}}">
+  <img src="_/img/history.svg" title="History of this page" alt="History" height="17">
+</a>
+<a class="page-icon" href="https://github.com/bonitasoft/bonita-documentation-site/blob/master/docs/content/CONTRIBUTING.adoc">
+  <img src="_/img/info.svg" title="How to contribute" alt="Contribution" height="17">
```

Comment:
❓ about the icons, are they from an orginal design you did? If not can you put in comment where you find them?
⚠️ We must ensure that they are free to use or they are released under a license that allows us to integrate them.

Reply:
I found them on the web, I checked thay didn't have a license and are free to use. Not sure it will be easy for me to find them again...